eng/pipelines/templates/jobs/auto-release.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+  branches:
+    exclude:
+      - '*'
+
+# avoid being triggered as part of CI check
+pr:
+  branches:
+    exclude:
+      - '*'
+
+schedules:
+  - cron: "0,30 1-9 * * *"
+    displayName: Daily release
+    branches:
+      include:
+        - main
+    always: true
+
+variables:
+  - group: Release Secrets for GitHub
```

Comment:
We generally try to avoid the variable group name in the yml file but instead just set it up in the DevOps UI.